.github/workflows/build-containers.yml

```diff
@@ -75,7 +75,7 @@ jobs:
             org.opencontainers.image.created=${{ steps.date.outputs.date }}
             org.opencontainers.image.title=${{ matrix.title }}
             org.opencontainers.image.description=${{ matrix.description }}
-            org.opencontainers.image.base.name=mcr.microsoft.com/dotnet/aspnet:8.0-jammy-chiseled-composite-extra
+            org.opencontainers.image.base.name=mcr.microsoft.com/dotnet/aspnet:10.0-jammy-noble-composite-extra
```

Comment:
To go to noble, we'd need to validate that this error is no longer an issue when hosting in Azure.

https://github.com/Particular/ServiceControl/pull/4422

I want to say that we reported to Microsoft and they fixed it, but can't remember for sure. @bording do you remember?

Reply:
yeah that was me screwing up some edits to the file